src/main/java/domain/ladder/Ladder.java

```diff
@@ -0,0 +1,46 @@
+package domain.ladder;
+
+import domain.ladder.attirbute.Direction;
+import util.DirectionGenerator;
+import domain.ladder.attirbute.Height;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Ladder {
+    private final Integer playerCount;
+    private final Height height;
+    private final List<LadderLeg> ladderLegs;
+
+    public Ladder(Height height, Integer playerCount, DirectionGenerator directionGenerator) {
+        this.height = height;
+        this.playerCount = playerCount;
+
+        this.ladderLegs = generateFirstToLastLegs(directionGenerator);
+    }
+
+    private List<LadderLeg> generateFirstToLastLegs(DirectionGenerator directionGenerator) {
+        LadderLegGenerator ladderLegGenerator = new LadderLegGenerator(this.height);
+
+        List<LadderLeg> ladderLegs = new ArrayList<>();
+        LadderLeg ladderLeg = ladderLegGenerator.generateDownLadderLeg();
+        for (int i = 0; i < this.playerCount - 1; i++) {
```

Comment:
```suggestion
        for (int i = 0; i < width; i++) {
```
`playerCount`를 여기서만 사용하고 있는데, `this.playerCount - 1`로 for문안에 이해가 필요한 로직을 담기보단 `playerCount`를 다른 네이밍으로 변경하고 `this.playerCount - 1`이 계산된 값을 들고 있어도 되지 않을까요? for문이 좀 더 깔끔해질 것 같네요.